jwst/cube_build/ifu_cube.py

```diff
@@ -1620,11 +1620,11 @@ def map_miri_pixel_to_sky(self, input_model, this_par1, subtract_background):
         # If it has not been subtracted and the background has not been
         # subtracted - subtract it.
         num_ch_bgk = len(input_model.meta.background.polynomial_info)
-        if(num_ch_bgk > 0 and subtract_background and input_model.meta.background.subtracted is False):
+        if (num_ch_bgk > 0 and subtract_background and input_model.meta.background.subtracted is False):
```

Comment:
```suggestion
        if num_ch_bgk and subtract_background and not input_model.meta.background.subtracted:
```
but double-check: can `input_model.meta.background.subtracted` be anything other than `True` or `False` (i.e., `None`).